### PR TITLE
More minor baydev fixes

### LINKED ISSUE #1235: Dev: Donut boxes do not display how many donuts they have on sprite
image

Invisible donuts on the test server.

### PR DESCRIPTION
**Pull request by Alberyk**
-fixed #1235
-fixed stunbaton being borked
-fixed wrong lobby screen
-re-added nanako's changes to embedding
-remove old mosin code
-fix a small issue with the body type proc on the human species
-fixes an issue with the mounted smg

### CODE REVIEW

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -107,57 +107,73 @@
 		return
 	return ..()
 
-/obj/item/weapon/melee/baton/apply_hit_effect(mob/living/target, mob/living/user, var/hit_zone)
```
Comment (NanakoAC) [inline code/game/objects/items/weapons/stunbaton.dm]: Can anyone think of anything bad that could result from not passing in the hit zone here?

I don't like the idea of having to retrieve it from the user within this function, it makes overriding more of a pain.
Is it removed upstream or just in this override?

Comment (Alberyk) [inline code/game/objects/items/weapons/stunbaton.dm]: I believe that the zone is checked later, also, I tested and it works as intended; if the baton is offline, no stun will be applied.

Comment (skull132) [inline code/game/objects/items/weapons/stunbaton.dm]: Nanako is actually kinda right about this.

`var/target_zone = check_zone(user.zone_sel.selecting)` can be replaced with `var/target_zone = check_zone(hit_zone)`, if you leave `var/hit_zone` in the proc definition. `user.zone_sel_selecting` is called in the initial call to `attack()` anyways.

**Comment by Alberyk** [comment]
Changes done.

**Pull request merged**